### DISCUSSION TITLE
lineY 画最大值，最小值，平均值时，标签text回调能不能把这个值抛出来

### DISCUSSION THREAD

**Discussion by xyr550** [antvis/G2#5261] [Q&A]
想实现如下效果

![image](https://github.com/antvis/G2/assets/73283715/3d807b33-5461-4e01-a2cf-c9966f430d3e)

![image](https://github.com/antvis/G2/assets/73283715/acce5d3e-9281-48f4-a227-034ef0e48104)

**Comment by pearmini** [comment]
确实是需要的。
